completion.sh

```diff
@@ -0,0 +1,83 @@
+# This file is part of the Symfony package.
```

Comment:
quel est ce fichier shell dans ce projet?

Reply:
Symfony documentation : [console]( https://symfony.com/doc/current/console.html )

"If you are using the Bash or Fish shell, you can install Symfony's completion script to get auto completion when typing commands in the terminal. All commands support name and option completion, and some can even complete values."

Command : `$ composer completion --help` 
Result :

 Dump the script to a global completion file and restart your shell:
  
      /usr/local/bin/composer completion bash | sudo tee /etc/bash_completion.d/composer
  
  Or dump the script to a local file and source it:
  
      /usr/local/bin/composer completion bash > completion.sh
  
      # source the file whenever you use the project
      source completion.sh
  
      # or add this line at the end of your "~/.bashrc" file:
      source /path/to/completion.sh